modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc

```diff
@@ -47,6 +47,29 @@ In the following example, the app will perform an `ADD_TASK` mutation which will
 
 For more information, see link:https://www.apollographql.com/docs/react/essentials/mutations.html#props[Apollo's document about mutations].
 
+=== Global Update Functions
+
+Apollo client holds all mutation parameters in memory.
+Offline Client will save and restore all mutations with exact parameters
+when application is restarted. Update functions supplied to mutations cannot be saved in the cache. 
+As result all optimisticResponses will disappear from application after restart.
+
+To prevent from that Client offers separate `mutationCacheUpdates` parameter that can be supplied to client config.
```

Comment:
"To prevent the loss of all _optimisticResponses_ after a restart, you can configure the _Update Functions_ to restore all _optimisticResponses_  once the  Apollo client comes back online:"